### PR TITLE
bumping vispy to non-dev 0.4.0 release

### PR DESCRIPTION
**Pull request by rsignell-usgs**

### CODE REVIEW

File: vispy/meta.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-    version: "0.4.0.dev"
+    version: "0.4.0"
 
 source:
     git_url: https://github.com/vispy/vispy.git
-    git_tag: master
+    git_tag: v0.4.0
 
 build:
     number: 0
```
Comment (ocefpaf) [inline vispy/meta.yaml]: Bump the build number!

**Comment by rsignell-usgs** [comment]
okay, bumped the build number from 0 to 1.  I wasn't sure I needed to do that: I thought conda would realize that 0.4.0 is an update to 0.4.0.dev

**Comment by ocefpaf** [comment]
Oops, you are right!  I did not notice that you changed from master to a release!  My bad :flushed: 

Merging it.

**Comment by rsignell-usgs** [comment]
Ack, the packages are there, but `conda install` thinks that "0.4.0.dev" is more up to date than "0.4.0".   I will delete the old ones.   We should remember this when we name dev versions.

**Comment by ocefpaf** [comment]
My bad. I need to pay more attention to https://www.python.org/dev/peps/pep-0440/

**Comment by rsignell-usgs** [comment]
@ocefpaf , dang it
```
binstar remove ioos/vispy/1.4.0.dev
```
isn't working.  Do you know the correct syntax?

**Comment by rsignell-usgs** [comment]
Nevermind, I removed using the GUI on binstar

**Comment by ocefpaf** [comment]
I was going to suggest that :grimacing:

**Pull request merged**